.github/workflows/deploy-nextjs-app-pr.yaml

```diff
@@ -0,0 +1,46 @@
+name: Deploy website from PR
+
+on:
+  pull_request:
+    branches: [main]
+    types: [labeled, edited]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.label.name == 'PR_DEPLOY' }}
+    defaults:
+      run:
+        working-directory: ./apps/next
+    env:
+      NODE_ENV: ci
```

Comment:
I feel like this will break a lot of things that depend on the common practice of NODE_ENV being  either production, development, or test. Why do you need to do this?

Reply:
I don't think I need it actually, especially now that I've deleted `deploy-nextjs-app-dev.yaml`.